PhysicsTools/PatAlgos/python/slimming/applyDeepBtagging_cff.py

```diff
@@ -47,6 +47,26 @@ def applyDeepBtagging( process, postfix="" ) :
     # update slimmed jets to include particle-based deep taggers (keep same name)
     # make clone for DeepTags-less slimmed AK8 jets, so output name is preserved
     addToProcessAndTask('slimmedJetsAK8NoDeepTags', process.slimmedJetsAK8.clone(), process, task)
+    _btagDiscriminators = cms.PSet( names = cms.vstring(
+        'pfDeepDoubleBJetTags:probQ',
+        'pfDeepDoubleBJetTags:probH' )
+    )
+    from Configuration.Eras.Modifier_run2_miniAOD_devel_cff import run2_miniAOD_devel
+    run2_miniAOD_devel.toModify(_btagDiscriminators, names = _btagDiscriminators.names + [
+            'pfDeepDoubleBvLJetTags:probQCD',
+            'pfDeepDoubleBvLJetTags:probHbb',
+            'pfDeepDoubleCvLJetTags:probQCD',
+            'pfDeepDoubleCvLJetTags:probHcc',
+            'pfDeepDoubleCvBJetTags:probHbb',
+            'pfDeepDoubleCvBJetTags:probHcc',
+            'pfMassIndependentDeepDoubleBvLJetTags:probQCD',
+            'pfMassIndependentDeepDoubleBvLJetTags:probHbb',
+            'pfMassIndependentDeepDoubleCvLJetTags:probQCD',
+            'pfMassIndependentDeepDoubleCvLJetTags:probHcc',
+            'pfMassIndependentDeepDoubleCvBJetTags:probHbb',
+            'pfMassIndependentDeepDoubleCvBJetTags:probHcc',
+            ] # + pfDeepBoostedJetTagsAll
```

Comment:
the goal for ```run2_miniAOD_devel```
was to keep synchronization with the master version of miniAOD wherever possible.
So, ```pfDeepBoostedJetTagsAll``` better be uncommented.